src/javascripts/components/blog/ArticlePreview.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+class ArticlePreview extends React.Component {
+  render() {
+    return (
+      <Link to="/article"><div className="articlePreview">
+        <div className="--imageContainer">
+          <img
+            src="./images/alice-in-wonderland.png"
+            alt="Picture of 'Author'"
+          />
```

Comment:
Also...  I don't think that's a picture of the author?  You might want to edit that `alt` text or just `alt=""` in this case